.claude/skills/deep-research/SKILL.md

```
@@ -0,0 +1,201 @@
+  5. VERIFY   — Run quality gate checklist (see below)
+  6. LOG      — Append round summary to results.jsonl
+  7. DECIDE   — Gate passed? → trigger verification subagent
+                 Gate failed? → continue to Round N+1
+                 Max iterations reached? → mark gaps + trigger verification
+```
+
+### Round 1 Bootstrap
+
+On the first round, create the state files:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)